api/index.js

```diff
@@ -321,6 +321,14 @@ app.use('/', (req, res) => {
   })
 });
 
+app.use((err, req, res, next) => {
+  err.message = `Failed to process ${req.path} request. ${err.message }`;
```

Comment:
Why do you need a space after `err.message`?

Reply:
After commas and dot I always put space. Try to imagine such a message which is not readable:
"Failed to process /some/path request.Some error"

In many editors it will be highlighted as an error.

Reply:
fixed